tripal/src/Services/TripalTokenParser.php

```diff
@@ -127,6 +127,13 @@ public function initParser(TripalEntityType $bundle, TripalEntity $entity = NULL
    *   The property values
    */
   public function addFieldValue($field_name, string $key, $value){
+    // Special case handling, fields set to one of the HTML formats
+    // will be stored wrapped in paragraph tags <p></p>.
+    // These are disruptive to the title, so remove them.
+    // Any other HTML markup is allowed, e.g. <em> <strong>
+    if ($value) {
+      $value = preg_replace('/^<p>(.*)<\/p>[\r\n]*$/', '$1', $value);
```

Comment:
Shouldn't this one also use the allowed tags and XSS approach?

Reply:
This change can be reverted, the &lt;p&gt; tags will now get filtered out by XSS::sanitize later.

Reply:
I'm always happy to get rid of "special case" handling code 🙃